txpool/txpool.go

```diff
@@ -345,16 +360,13 @@ func (t *TxPool) addImpl(origin TxOrigin, tx *types.Transaction) error {
 		t.locals.addAddr(tx.From)
 	}
 
-	// Skip check of GasPrice in the future transactions created by same address when TxPool receives transaction by Gossip or Reorg
-	if isLocal && !t.locals.containsAddr(tx.From) {
-		t.locals.addAddr(tx.From)
-	}
-
+	// Move promotable txs to the pending queue
```

Comment:
Not saying it is wrong, but why did we get rid of this locals functionality?

Reply:
We didn't. It was duplicate code.